src/apiClient.js

```diff
@@ -15,24 +13,8 @@ export default function APIClient(mpInstance, kitBlocker) {
             this.uploader = new BatchUploader(mpInstance, millis);
         }
         this.uploader.queueEvent(event);
-    };
-
-    this.shouldEnableBatching = function() {
```

Comment:
Not needed anymore since the flags are all 100 in the database now.  A to do for the future would be to remove the feature flag from being sent down from the server to save on size further.